### ISSUE TITLE
Needs new Git version, should support older Git versions

### ISSUE THREAD

**Issue by grandchild** [hartwork/git-delete-merged-branches#12]
Git version on Ubuntu 18 is still 2.17.1

Fails with
```
$ cd my/project
$ git-delete-merged-branches
error: unknown option `show-current'
usage: [...]
Error: Command 'git branch --format=%(refname:lstrip=2) --show-current' returned non-zero exit status 129.
$ git --version
git version 2.17.1
```

**Comment by grandchild** [comment]
`--show-current` available in 2.20.0

https://github.com/git/git/blob/74583d89127e21255c12dd3c8a3bf60b497d7d03/Documentation/RelNotes/2.22.0.txt

**Comment by hartwork** [comment]
That's sad but definitely a valuable report — thank you! :+1:

**Comment by hartwork** [comment]
Fixed in release 3.1.0 by commit 9e437d2f3fe4e53acf3663806058f8c6e01edbf4, closing.
